Common/include/option_structure.hpp

```diff
@@ -64,12 +64,12 @@ using MapType = CEmptyMap<T,U>;
  * \brief Different software components of SU2
  */
 enum SU2_COMPONENT {
-  SU2_CFD = 1,  /*!< \brief Running the SU2_CFD software. */
-  SU2_DEF = 2,  /*!< \brief Running the SU2_DEF software. */
-  SU2_DOT = 3,  /*!< \brief Running the SU2_DOT software. */
-  SU2_MSH = 4,  /*!< \brief Running the SU2_MSH software. */
-  SU2_GEO = 5,  /*!< \brief Running the SU2_GEO software. */
-  SU2_SOL = 6   /*!< \brief Running the SU2_SOL software. */
+  SU2_CFD = 1,   /*!< \brief Running the SU2_CFD software. */
+  SU2_DEF = 2,   /*!< \brief Running the SU2_DEF software. */
+  SU2_DOT = 3,   /*!< \brief Running the SU2_DOT software. */
+  SU2_MSH = 4,   /*!< \brief Running the SU2_MSH software. */
+  SU2_GEO = 5,   /*!< \brief Running the SU2_GEO software. */
```

Comment:
Hmmm I deleted SU2_MSH from here recently... Delete it again please, and don't add extra spaces for no reason, that tends to create merge conflicts and that is how we end up re-adding things that were removed.

Reply:
@pcarruscag SU2_MSH is still there, on the last commit of develop

Reply:
Because someone put it back, probably because of SU2 interp, the code itself does not exist.

Reply:
My bad, I removed it from some python files.